modules/round/src/main/JsonView.scala

```diff
@@ -356,10 +356,10 @@ object JsonView {
     import lila.common.Maths.truncateAt
     Json.obj(
       "running" -> c.isRunning,
-      "initial" -> c.limit,
-      "increment" -> c.increment,
-      "white" -> truncateAt(c.remainingTime(Color.White), 2),
-      "black" -> truncateAt(c.remainingTime(Color.Black), 2),
+      "initial" -> c.limitSeconds,
+      "increment" -> c.incrementSeconds,
+      "white" -> c.remainingTime(Color.White).toSeconds,
+      "black" -> c.remainingTime(Color.Black).toSeconds,
```

Comment:
I think we get the same behavior, although it's possible the float won't represent the 0 and will get written verbosely.  Safer would be to use this method from Clock:

`private val limitFormatter = new DecimalFormat("#.##")`